### ISSUE TITLE
Uncaught TypeError: canvas is null

### ISSUE THREAD

**Issue by Ala-Na** [Reaster0/ft_transcendence#71]
I encounter this error inside `Console` each time I resize my window.
Funny enough it doesn't pop when on "home page", but once I clicked on some buttons (account/game/...), it's unavoidable.

The details of the error sometimes changes, so here are some logs of it:
`Uncaught TypeError: canvas is null
    resize CanvasComp.vue:44
    resize CanvasComp.vue:44
    resize self-hosted:1178
    resize self-hosted:1178`

`Uncaught TypeError: canvas is null
    resize CanvasComp.vue:44
    mounted CanvasComp.vue:76
    callWithErrorHandling runtime-core.esm-bundler.js:155
    callWithAsyncErrorHandling runtime-core.esm-bundler.js:164
    __weh runtime-core.esm-bundler.js:2685
    flushPostFlushCbs runtime-core.esm-bundler.js:356
    flushJobs runtime-core.esm-bundler.js:401`

`Uncaught TypeError: canvas is null
    resize CanvasComp.vue:44
    mounted CanvasComp.vue:76
    callWithErrorHandling runtime-core.esm-bundler.js:155
    callWithAsyncErrorHandling runtime-core.esm-bundler.js:164
    __weh runtime-core.esm-bundler.js:2685
    flushPostFlushCbs runtime-core.esm-bundler.js:356
    flushJobs runtime-core.esm-bundler.js:401
    promise callback*queueFlush runtime-core.esm-bundler.js:285
    queueJob runtime-core.esm-bundler.js:279
    effect runtime-core.esm-bundler.js:5168
    triggerEffects reactivity.esm-bundler.js:382
    triggerRefValue reactivity.esm-bundler.js:982
    effect reactivity.esm-bundler.js:1113
    triggerEffects reactivity.esm-bundler.js:382
    triggerRefValue reactivity.esm-bundler.js:982
    effect reactivity.esm-bundler.js:1113
    triggerEffects reactivity.esm-bundler.js:382
    triggerRefValue reactivity.esm-bundler.js:982
    effect reactivity.esm-bundler.js:1113
    triggerEffects reactivity.esm-bundler.js:382
    triggerRefValue reactivity.esm-bundler.js:982
    effect reactivity.esm-bundler.js:1113
    triggerEffects reactivity.esm-bundler.js:382
    triggerRefValue reactivity.esm-bundler.js:982
    set value reactivity.esm-bundler.js:1026
    finalizeNavigation vue-router.esm-bundler.js:3197
    pushWithRedirect vue-router.esm-bundler.js:3070
    promise callback*pushWithRedirect vue-router.esm-bundler.js:3040
    push vue-router.esm-bundler.js:2969
    install vue-router.esm-bundler.js:3386
    use runtime-core.esm-bundler.js:3834
    <anonymous> main.js:10
    js app.js:184
    __webpack_require__ app.js:1470
    __webpack_exports__ app.js:2681
    O app.js:1516
    <anonymous> app.js:2682
    <anonymous> app.js:2684`

More info about this error (link associated with errors messages by console): https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Unexpected_type

Idk if someone was already aware of this error, but I'm making an issue of it just in case ;)

**Comment by Reaster0** [comment]
yeah i know what it is, it will be fix (one day lmao) no but seriously it's not a serious problem
